openapi.yml

```
@@ -282,7 +282,8 @@ paths:
       requestBody:
         description: OCRD-ZIP of the new workspace
         content:
-          'application/vnd.ocrd+zip': {}
+          'application/vnd.ocrd+zip':
+            schema: {$ref: '#/components/schemas/WorkspaceRequest'}
```

If you really don't want the upload to be a multi-part form (descriptive JSON + binary workspace), then I suggest to get rid of the `WorkspaceRequest` altogether and instead make this a simple file upload.

```suggestion
          'application/vnd.ocrd+zip':
            schema:
              type: string
              format: binary
```

BTW, you can upload the OpenAPI YAML definition to https://app.swaggerhub.com and edit interactively to see how the constructs would be rendered by the Swagger UI. I have adapted https://app.swaggerhub.com/apis/kba/ocr-d_web_api/0.0.1#/workspace/createWorkspace to show how this would be rendered. If you click on the "Try it Out" button, you should see a file upload form element.